Package.swift

```diff
@@ -1359,6 +1359,7 @@ func abseilDependency() -> Package.Dependency {
       "https://github.com/google/abseil-cpp-binary.git",
       "1.2024011601.1" ..< "1.2024011700.0"
     )
+    return .package(url: packageInfo.url, branch: "nc/firestore-pre_rc1")
```

Comment:
Reminder to raise minimum supported version for Abseil. Probably should do this in grpc-binary as well.